src/components/Login.js

```diff
@@ -1,12 +1,12 @@
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import { Link } from 'react-router-dom'
-import { postLogin } from '../actions/login'
-import { Form, Grid, Image, Divider, Icon, Message, Button } from 'semantic-ui-react'
-import PropTypes from 'prop-types'
-import login from './../styles/Login.css'
-import orgLogo from '../assets/org-logo.jpg'
-import { register } from '../urls'
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { postLogin } from '../actions/login';
+import { Form, Grid, Image, Divider, Icon, Message, Button } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+// import login from './../styles/Login.css'
```

Comment:
is this not needed?

Reply:
The import login wasn't used anywhere in Login.js file which resulted in the warning "Used of undeclared variable login".
That's why we need to comment out that.

Reply:
Have you tried rewriting it like this `import './../styles/Login.css'` ?

Reply:
No